### PR TITLE
Converted remainder of external links

### PR DESCRIPTION
**Pull request by joolswood**
## Description

Styles all remaining external links as external links.

### CODE REVIEW

**Comment by joolswood** [comment]
This should close #5

File: _content-types/documents.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ If there is a user need for a PDF, [make the PDF accessible](/accessibility-incl
 
 Make Word documents accessible to everyone even if you are emailing them internally. This includes people who rely on assistive technologies, such as screen readers.
 
-Microsoft has guidance on [making Word documents more accessible ](https://support.office.com/en-us/article/Creating-accessible-Word-documents-d9bf3683-87ac-47ea-b91a-78dcacb3c66d?CTT=3&CorrelationId=b3c40249-a7fa-4300-9c12-5d0f79242a78&ui=en-US&rs=en-US&ad=US).
+Microsoft has guidance on <a href="https://support.office.com/en-us/article/Creating-accessible-Word-documents-d9bf3683-87ac-47ea-b91a-78dcacb3c66d?CTT=3&amp;CorrelationId=b3c40249-a7fa-4300-9c12-5d0f79242a78&amp;ui=en-US&amp;rs=en-US&amp;ad=US" rel="external">making Word documents more accessible </a>.
```
Comment (bjgordon) [inline _content-types/documents.md]: I think you can actually leave these as markdown if that makes it easier I'll double check but I think it's  `[link text] (url){:rel => external}`

Comment (bjgordon) [inline _content-types/documents.md]: FYI the correct way is to append {:rel="external"}.

Comment (joolswood) [inline _content-types/documents.md]: Can you link me to any doco on `{:rel="external"}`? Just did a quick search and I couldn't find anything.

Comment (bjgordon) [inline _content-types/documents.md]: It's kramdown specific: https://kramdown.gettalong.org/quickref.html#block-attributes

**Comment by joolswood** [comment]
Just did a test @bjgordon and the kramdown version is good. Let's go with that. I tried to do a Regex search and find but Atom doesn't except `^` and `$` so that makes it a bit too hard. I can do it manually, unless you can figure out an automation? Maybe with vim?

**Comment by bjgordon** [comment]
@joolswood I've done a regex find/replace to change the `<a ...rel="external">` back to markdown. No warranty provided, but it looks ok to me.

**Comment by joolswood** [comment]
LGTM and mergin' as soon as test is run

**Pull request merged**